tests/datasets/test_array_dataset.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import pytest
+import torch
+from pytorch_lightning.utilities import exceptions
+
+from pl_bolts.datasets import ArrayDataset, DataModel
+from pl_bolts.datasets.utils import to_tensor
+
+
+def add_one(integers: np.ndarray) -> np.ndarray:
```

Comment:
Type it with `ARRAYS`, since it can accept anything.

Reply:
And it becomes a bit tricky, if it should accept all three types of `ARRAYS` (Tensors, NumPy Arrays and regular lists). You could maybe play with something like `functools.single_dispatch`

Reply:
Btw, these comments apply to all these auxiliary functions for testing

Reply:
I have decided that it is just easier to use the `to_tensor` function for testing 😅